Frontend/ui-customer/src/Flow.purs

```diff
@@ -2528,10 +2532,17 @@ fetchAndModifyLocationLists savedLocationResp = do
               , locationList = updatedLocationList
               , destinationSuggestions = updatedLocationList
               , suggestionsData{suggestionsMap = suggestionsMap}
-              , tripSuggestions = sortedTripList
+              , tripSuggestions = removeDuplicateTrips sortedTripList 
               }
             })
     where
+
+      removeDuplicateTrips :: Array Trip -> Array Trip
+      removeDuplicateTrips trips = 
+        let grouped = groupBy (\a b -> (getGeoHash a.destLat a.destLong 9) == (getGeoHash b.destLat b.destLong 9)) trips
```

Comment:
@aryan-13 instead of a, b can you give better variable name

Reply:
also why Geohash precision is 9, on what basis this is decided ?

Reply:
Can we make geohash precision configurable

Reply:
So as to remove the trips with destinations within 4.8x4.8m block.